hoomd/update/box_resize.py

```diff
@@ -0,0 +1,35 @@
+from hoomd.operation import _Updater
+from hoomd.box import Box
+from hoomd.parameterdicts import ParameterDict
+from hoomd.typeconverter import OnlyType
+from hoomd.variant import Variant
+from hoomd.util import variant_preprocessing
+from hoomd._hoomd import BoxResizeUpdater
+
+
+class BoxResize(_Updater):
+    def __init__(self, box1, box2, variant, trigger):
+        params = ParameterDict(
+            box1=OnlyType(Box), box2=OnlyType(Box),
+            variant=OnlyType(Variant, variant_preprocessing),
+            scale_particles=True)
```

Comment:
I think that would be fine. I just went with the previous behavior since I think that `True` is a good default, but that could just be an optional argument.